### ISSUE TITLE
Updating records inline is not working

### ISSUE THREAD

**Issue by steebchen** [isaac-mcfadyen/d1-jdbc-driver#6]
First, thanks a lot for this project, whatever is available is already amazing to have and is an extreme productivity booster when working with D1 ❤️. Maybe enable GitHub sponsorships?

For the actual issue, just wanted to leave this here for tracking, my knowledge with java or IJ plugins is basically non-existent.

Basically when editing something in a result table from DataGrip and then updating a value does not work as it produces an incompatible value:

<img width="757" alt="image" src="https://github.com/isaac-mcfadyen/d1-jdbc-driver/assets/5013932/b5f55c41-8f66-41a1-bc38-eb2927de4705">

This results in an error:

```
prepareStatement(...) must not be null
```

Instead of saving the changes, I can let it generate the equivalent SQL statement, which is the following:

```sql
UPDATE product t
SET t.title = null, t.slug = null
WHERE t.id LIKE 'kd891h' ESCAPE '#';
```

Unfortunately, when executing this, it also results in an error "invalid query".

The fix for this one at least is easy by removing the alias:

```sql
UPDATE product
SET title = null, slug = null
WHERE id LIKE 'kd891h' ESCAPE '#';
```

and then running it in the db console instead. This will be my workaround for now which is good enough :)

**Comment by Auios** [comment]
Getting this issue as well. I took a look through the source code to try and figure out how the alias was being generated. I've not got any previous experience working with JDBC drivers so this was all a foreign experience for me. Couldn't find anything in there about the alias. Anyone know anything?

I guess this issue I made is a duplicate of this issue?
https://github.com/isaac-mcfadyen/d1-jdbc-driver/issues/9

**Comment by Shannor** [comment]
Yeah, bump on this, it's still an issue and haven't heard much about it yet.
